docs/lineage/lineage-feature-guide.md

```diff
@@ -0,0 +1,129 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Lineage
+
+<FeatureAvailability/>
+
+In DataHub, lineage is how describe the way that data flows within and between your source systems. For a given entity, lineage allows you to see where this data is coming from (what's upstream) and where it's going (what's downstream). For more information, see [this video](https://www.youtube.com/watch?v=rONGpsndzRw&ab_channel=DataHub) for Lineage 101 in DataHub.
+
+Starting in version `0.9.5`, DataHub supports the manual editing of lineage between entities. Data experts are free to add or remove upstream and downstream lineage edges in both the Lineage Visualization screen as well as the Lineage tab on entity pages. Use this feature to supplement automatic lineage extraction in ingestion or establish important entity relationships in sources that we don't support automatic extraction for yet! Editing lineage by hand is only supported for Datasets, Charts, Dashboards, and Data Jobs for now.
+
+:::note
+
+Lineage added by hand and programmatically may conflict with one another to cause unwanted overwrites. It is strongly recommend that lineage is edited manually in cases where lineage information is not also extracted in automated fashion, e.g. by running an ingestion source.
+
+:::
+
+## Manual Lineage Setup, Prerequisites, and Permissions
+
+To edit lineage for an entity, you'll need the following [Metadata Privilege](../authorization/policies.md):
+
+* **Edit Lineage** metadata privilege to edit lineage at the entity level
+
+It is important to know that the **Edit Lineage** privilege is required for all entities whose lineage is affected by the changes. For example, in order to add "Dataset B" as an upstream dependency of "Dataset A", you'll need the **Edit Lineage** privilege for both Dataset A and Dataset B.
+
+## Using Manual Lineage
+
+### Editing from Lineage Graph View
+
+The first place that you can edit lineage for entities is from the Lineage Visualization screen. Click on the "Lineage" button on the top right of an entity's profile to get to this view.
+
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/lineage/lineage-viz-button.png"/>
+</p>
+
+Once you find the entity that you want to edit the lineage of, click on the three-dot menu dropdown to select whether you want to edit lineage in the upstream direction or the downstream direction.
+
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/lineage/edit-lineage-menu.png"/>
+</p>
+
+If you want to edit upstream lineage for entities downstream of the center node or downstream lineage for entities upstream of the center node, you can simply re-center to focus on the node you want to edit. Once focused on the desired node, you can edit lineage in either direction.
```

Comment:
can we insert a comma after "entities downstream of the center nodes"?

Reply:
hm I actually don't agree with this one since there's only two things here that are sharing the same verb "edit" - maybe if you think this is hard to read I can make it:
```
If you want to edit upstream lineage for entities downstream of the center node, or edit downstream lineage for entities upstream of the center node, you can simply
```
and repeat "edit"?
I'm referencing this grammar website as well lol https://www.grammarly.com/blog/comma-before-or/?gclid=CjwKCAiA8OmdBhAgEiwAShr405I0ZWHmNMwt3HgrXCMW2hsJZqDQMn9MBfweTcC9YE24IsS-IvVxNRoCSl0QAvD_BwE&gclsrc=aw.ds